docs/api/browser-window.md

```diff
@@ -615,6 +615,19 @@ Returns:
 
 Emitted on 3-finger swipe. Possible directions are `up`, `right`, `down`, `left`.
 
+#### Event: 'rotate-gesture' _macOS_
+
+Returns:
+
+* `event` Event
+* `rotation` Float
+
+Emitted on trackpad rotation gesture. Continually emitted until rotation gesture is
+ended. The `rotation` value on each emission is the angle in degrees rotated since
+the last emission. The last emitted event upon a rotation gesture will always be of
```

Comment:
Point of clarification -- is this the delta since the last emission, or the delta since the rotation began? These docs seem to be saying the former, but online docs of NSEvent seem to be saying the latter.

I'm not asserting this is correct or incorrect; I haven't tested for myself. But if this is the delta since rotation began, we should say that instead.

Reply:
@ckerr I found the official docs to be confusing in this regard. From my testing, it seemed like the emitted value on each tick was the Δdegrees since the last tick. I tried doing a 90-degree rotation, and the value from each emitted event added up to more or less 90 degrees.